### PR TITLE
#155 - Implemented

### PR DESCRIPTION
**Pull request by aaguiar96**
Hi @miguelriemoliveira  and @eupedrosa 

This is implemented and working. Can you check if the code is suitable to integrate in atom?

### CODE REVIEW

File: atom_core/src/atom_core/utilities.py [hunk 1]
```diff
@@ -102,12 +107,14 @@ def execute(cmd, blocking=True, verbose=True):
         @return
     """
     if verbose:
-        print "Executing command: " + cmd
+        print
```
[the remark this answers is not in the document]
Comment (aaguiar96) [reply atom_core/src/atom_core/utilities.py]: Hi @eupedrosa 

This is fixed. I do not know how it happened, but is fixed!
Can you merge and test please?

**Comment by miguelriemoliveira** [comment]
If @eupedrosa approved, you have my blind vote : - )

**Comment by eupedrosa** [comment]
> If @eupedrosa approved, you have my blind vote : - )

You should not have..
This break the existing datasets.

**Comment by miguelriemoliveira** [comment]
Yes, just realized it now ... https://github.com/lardemua/atom/issues/155
*  Miguel Riem de Oliveira                                             *
email:    mriem@ua.pt    |    m.riem.oliveira@gmail.com
office: 22.2.18                 |    ext: 23887
Dep. de Engenharia Mecânica, Universidade de Aveiro
*Campus Universitário de Santiago, 3810-193  Portugal*

**Pull request merged**